test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/Script.cshtml

```diff
@@ -11,20 +11,59 @@
 </head>
 <body>
     <h2>Script tag helper test</h2>
-    <script src="~/blank.js" data-foo="foo-data1">
+    <script src="~/site.js" data-foo="foo-data1">
         // Regular script with comment in body, and extra properties.
     </script>
 
-    <script src="~/blank.js" asp-fallback-src="~/fallback.js" asp-fallback-test="false" data-foo="foo-data2">
+    <script src="~/blank.js" asp-fallback-src="~/site.js" asp-fallback-test="false" data-foo="foo-data2">
         // TagHelper script with comment in body, and extra properties.
     </script>
 
-    <script asp-fallback-src="~/fallback.js" asp-fallback-test="false" data-foo="foo-data3">
+    <script src="~/blank.js" asp-fallback-src-include="**/*.js" asp-fallback-src-exclude="**/site3.js" asp-fallback-test="false">
```

Comment:
Would like to see more variations of just the `asp-fallback-src-include`/`asp-fallback-src-exclude` bits, all other attributes shave multiple versions.